misk/src/main/kotlin/misk/clustering/kubernetes/KubernetesConfig.kt

```diff
@@ -2,10 +2,17 @@ package misk.clustering.kubernetes
 
 import misk.config.Config
 
+/**
+ * @property clustering_pod_label_selector Optional Kubernetes label selector to filter which pods
+ *                                         in the namespace are considered to be in the same cluster.
+ *                                         If omitted, all healthy pods in the namespace are included in the cluster.
+ *                                         Ex: "app = helloserver".
+ */
 data class KubernetesConfig(
   val my_pod_namespace: String = System.getenv("MY_POD_NAMESPACE") ?: "<invalid-namespace>",
   val my_pod_name: String = System.getenv("MY_POD_NAME") ?: "<invalid-pod-name>",
   val my_pod_ip: String = System.getenv("MY_POD_IP") ?: "<invalid-pod-ip>",
+  val clustering_pod_label_selector: String? = null,
```

Comment:
should this default to `"app = $my_pod_namespace"` instead of null?

Reply:
I put the opinionated default in `skim`, since having a misk default that doesn't just work feels surprising.